client/lxd_images.go

```diff
@@ -421,9 +421,12 @@ func (r *ProtocolLXD) CreateImage(image api.ImagesPost, args *ImageCreateArgs) (
 		w := multipart.NewWriter(pw)
 
 		go func() {
+			var ioErr error
 			defer func() {
-				w.Close()
-				pw.Close()
+				if cerr := w.Close(); ioErr == nil && cerr != nil {
+					ioErr = cerr
+				}
+				_ = pw.CloseWithError(ioErr)
```

Comment:
This is falling foul of our static analysis rules:

```
Checking that functional blocks are followed by newlines...
client/lxd_images.go:429
```

Reply:
Fixed!